playground/src/components/base_info_header.tsx

```diff
@@ -11,6 +17,8 @@ type BaseInfoHeaderProps = {
   onChangeAthenaSecret: (value: string) => void;
   module: ModuleMeta | undefined;
   onChangeModule: (value: ModuleMeta) => void;
+  moduleConfig: any | undefined;
```

Comment:
```suggestion
  moduleConfig: any;
```
To my knowledge, `any` already allows for `undefined` values.

Reply:
Yeah, I like the explicit undefined but I removed it everywhere now.